prow/cmd/entrypoint/main.go

```diff
@@ -25,9 +28,61 @@ import (
 	"k8s.io/test-infra/prow/pod-utils/options"
 )
 
+const (
+	copyModeOnlyEnvVar = "COPY_BINARY_ONLY"
+
+	defaultCopyDst = "/tools/entrypoint"
+)
+
+var defaultCopySrc = []string{"/ko-app/entrypoint", "/entrypoint"}
```

Comment:
Shouldn't we be able to use `os.Args[0]` for the source location (and derefernece it if needed, IIRC bazel gives you a softlink as entrypoint)?

Reply:
actually I was wrong, os.Args[0] is the path of the symlink

Reply:
but the copy function is smart enough to copy the real binary